src/screens/HomeScreen/HomeScreen.js

```diff
@@ -1,19 +1,101 @@
 import React from 'react';
-import { View, Text, SafeAreaView } from 'react-native';
-import styles from './styles';
-import HeaderBar from '../../components/HeaderBar/HeaderBar';
+import { firebase } from '../../config/config';
+import { View, Text, SafeAreaView, Image, TouchableOpacity, FlatList } from 'react-native';
+import { withNavigation } from 'react-navigation';
 import { LOGO } from '../../config/styles.js';
+import styles from './styles';
+
+class HomeScreen extends React.Component{
+    state = {querydetail : []};
+    componentDidMount() {
+        const {querydetail} = this.state
+        firebase.firestore().collection("queries").orderBy('postdatetime', 'desc').onSnapshot((querySnapshot) => {
+            var newentity = []
+            var queryArray = []
+            querySnapshot.forEach((doc) => {
+                q_data = {
+                    "id": doc.data().id,
+                    "QueryInput": doc.data().QueryInput,
+                    "postTime": doc.data().postTime,
+                    "q_id": doc.id,
+                    "postdatetime": doc.data().postdatetime,
+                    "answer_present": doc.data().answer_present,
+                    "q_image" : doc.data().query_image
```

Comment:
Can you see the issue here? Use one single way when declaring variables. Do not mix different methods as this will make other devs hard to understand the code .

Reply:
Let me know if I changed the names correctly?